omnibus/config/projects/puppy.rb

```diff
@@ -124,6 +124,12 @@
 if windows?
   dependency 'datadog-agent-finalize'
 end
+
+# System probe
+if linux?
+  dependency 'system-probe'
+end
+
```

Comment:
I'm not too familiar with the puppy build, but is it an issue for the process-agent to be missing ?

Reply:
We don't include trace nor processes in puppy, so following that we shouldn't include system-probe either?

Reply:
Good catch. I’ve removed `system-probe` from the puppy build.